aibolit/patterns/multiple_while/multiple_while.py

```diff
@@ -0,0 +1,44 @@
+import javalang
+
+from collections import defaultdict
+import hashlib
+import itertools
+
+
+class MultipleWhile:
+
+    def __init__(self):
+        pass
+
+    def __file_to_ast(self, filename: str) -> javalang.ast.Node:
```

Comment:
@paulodamaso Please merge with upstream and use `AST `class instead of this method